packages/fluentui/react-northstar/src/components/Card/Card.tsx

```diff
@@ -87,7 +104,14 @@ const Card: React.FC<WithAsProp<CardProps>> &
       performClick: e => {
         handleClick(e);
       },
+      focusCell: e => {
+        e.preventDefault();
```

Comment:
Should keyboard event be prevented there?

Reply:
It should not, good catch ;)